### DISCUSSION TITLE
Unable to connect interface to multiple devices

### DISCUSSION THREAD

**Discussion by elixdreamer** [netbox-community/netbox#15488] [Help Wanted!]
### Deployment Type

NetBox Cloud

### NetBox Version

v3.7.4

### Python Version

3.8

### Steps to Reproduce

I'm trying to connect a single interface to multiple devices like [commented](https://github.com/netbox-community/netbox/issues/13923#issuecomment-1746435104) in this post on the demo site and this works fine unless you are using patch panels or Front and Rear ports inbetween.
![Fanout](https://github.com/netbox-community/netbox/assets/12197783/e688d99d-e1c3-49ce-9bc3-1501c9a95863)
This limits the posibilities for an E2E view on brakeout cables.

Can 

### Expected Behavior

Connections over multiple patchpanels and to multiple devices should be shown in the path trace

### Observed Behavior

Asymmetric Path! is displayed and you are unable to view the E2E patch using a breakoutcable

**Comment by craized** [comment]
Being able to connect rear ports to multiple devices is crucial for accurately modeling WDM systems that include combined MUX/DEMUX devices and separate pre/booster amps. I've already [commented](https://github.com/netbox-community/netbox/issues/13923#issuecomment-1739449627) as much but am bringing this here for proper discussion. 

I believe the below diagram fully describes the scenario

![image](https://github.com/netbox-community/netbox/assets/499818/8094fe77-397e-4025-8671-72543a0de7e6)
